internal/utils/jobhelper.go

```diff
@@ -15,154 +15,154 @@ import (
 type Status string
 
 const (
-	JobTypeBuild = "build"
-	JobTypeSign  = "sign"
+	PodTypeBuild = "build"
+	PodTypeSign  = "sign"
 
 	StatusCompleted  = "completed"
 	StatusCreated    = "created"
 	StatusInProgress = "in progress"
 	StatusFailed     = "failed"
 )
 
-var ErrNoMatchingJob = errors.New("no matching job")
+var ErrNoMatchingPod = errors.New("no matching pod")
 
 //go:generate mockgen -source=jobhelper.go -package=utils -destination=mock_jobhelper.go
 
-type JobHelper interface {
-	IsJobChanged(existingJob *batchv1.Job, newJob *batchv1.Job) (bool, error)
-	JobLabels(modName string, targetKernel string, jobType string) map[string]string
-	GetModuleJobByKernel(ctx context.Context, modName, namespace, targetKernel, jobType string, owner metav1.Object) (*batchv1.Job, error)
-	GetModuleJobs(ctx context.Context, modName, namespace, jobType string, owner metav1.Object) ([]batchv1.Job, error)
-	DeleteJob(ctx context.Context, job *batchv1.Job) error
-	CreateJob(ctx context.Context, jobTemplate *batchv1.Job) error
-	GetJobStatus(job *batchv1.Job) (Status, error)
+type PodHelper interface {
+	IsPodChanged(existingPod *v1.Pod, newPod *v1.Pod) (bool, error)
+	PodLabels(modName string, targetKernel string, podType string) map[string]string
+	GetModulePodByKernel(ctx context.Context, modName, namespace, targetKernel, podType string, owner metav1.Object) (*v1.Pod, error)
+	GetModulePods(ctx context.Context, modName, namespace, podType string, owner metav1.Object) ([]v1.Pod, error)
+	DeletePod(ctx context.Context, pod *v1.Pod) error
+	CreatePod(ctx context.Context, podSpec *v1.Pod) error
+	GetPodStatus(pod *v1.Pod) (Status, error)
 }
 
-type jobHelper struct {
+type podHelper struct {
 	client client.Client
 }
 
-func NewJobHelper(client client.Client) JobHelper {
-	return &jobHelper{
+func NewPodHelper(client client.Client) PodHelper {
+	return &podHelper{
 		client: client,
 	}
 }
 
-func (jh *jobHelper) IsJobChanged(existingJob *batchv1.Job, newJob *batchv1.Job) (bool, error) {
-	existingAnnotations := existingJob.GetAnnotations()
-	newAnnotations := newJob.GetAnnotations()
+func (jh *podHelper) IsPodChanged(existingPod *v1.Pod, newPod *v1.Pod) (bool, error) {
```

Comment:
```suggestion
func (ph *podHelper) IsPodChanged(existingPod *v1.Pod, newPod *v1.Pod) (bool, error) {
```
Please rename all receivers in this file.

Reply:
Thanks. Fixed.